test/lib/e2e-utils.ts

```diff
@@ -46,7 +46,9 @@ if (testModeFromFile === 'e2e') {
   const validE2EModes = ['dev', 'start', 'deploy']
 
   if (!process.env.NEXT_TEST_JOB && !testMode) {
-    console.warn('Warn: no NEXT_TEST_MODE set, using default of start')
+    require('console').warn(
```

Comment:
Whats the purpose of require() here?

Reply:
it bypasses jest delaying the logs from being emitted, x-ref: https://github.com/vercel/next.js/pull/35507